sci-electronics/kicad/kicad-9999.ebuild

```diff
@@ -25,8 +25,7 @@ else
 	fi
 fi
 
-# BSD for bundled pybind
```

Comment:
On a future bump, I'd strongly recommend adding a comment describing the licence of kicad, then listing the various bundled libraries with their licenses, so it's easy to keep it up-to-date and correct.